examples/platforms/nrf52840/README.md

```diff
@@ -53,17 +65,17 @@ The driver can be found in `third_party/NordicSemiconductor/libraries/usb/nordic
 
 You can build the libraries with support for USB bootloader with automatic USB DFU trigger support in PCA10059. As this dongle uses the native USB support, you must enable it as well.
```

Comment:
You can build the libraries with support for the USB bootloader with USB DFU trigger support in PCA10059.